tests/test_packaging_shell.py

```
@@ -15,8 +15,34 @@ def test_plugin_entry_point_loads_bare_module_without_side_effects() -> None:
+    # through a tiny context recorder; no SDK dependency is needed for this
+    # step because the factory must remain unconstructed.
+    try:
+        import agent.runtime_api  # noqa: F401
+    except ModuleNotFoundError:
+        return
+
+    class Context:
+        registration = None
+        provider_profile = None
+
+        def register_provider_profile(self, profile):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `register_provider_profile`

Add return type annotation: `None`

(ANN202)